### ISSUE TITLE
Suggestion: percentage at beginning of title

### ISSUE THREAD

**Issue by javl** [pwnbus/OctoPrint-Progress-Title#3]
Before I dive into learning how Octoprint plugins work to make this change I thought I'd check if this is something you could easily add ;)

Currently the title of the page is something like ```Printername [OctoPrint] - 50%```, but with multiple tabs open the browser only shows the first part of the page title. This means I only get to see the printer name.

Would it be possible to add a setting to toggle the location of the percentage? I'd love to move it to the front, so: ```50% - Printername [OctoPrint]```

I've updated my ```progress_title.js``` and swapped the ```self.default_title``` and the ```progress``` around, but it might be nice to have it as an option in the plugin itself.

```
self.updateProgress = function(progress) {
    document.title = progress + "% - " + self.default_title;
};
```

**Comment by pwnbus** [comment]
Thanks for the great idea, I will try and see if I can add support for that!

**Comment by javl** [comment]
That would be great. I guess a simple toggle to switch between percent at the start or at the end would do?

Here is an example of why I like the percentage at the start ;)
![image](https://user-images.githubusercontent.com/734644/80804972-078a4680-8bb7-11ea-92ad-d73cb387954a.png)

**Comment by xia0** [comment]
Agreed. This is a wonderful suggestion.

**Comment by MarcinOrlowski** [comment]
@pwnbus hi, any progress with that feature?

**Comment by SillyHats** [comment]
I agree. This would be a great addition to the plugin.

**Comment by pwnbus** [comment]
This is a bit delayed, but finally got around to adding a settings page to control where to put the percentage!
Available in v0.1.2
